README.md

```diff
@@ -62,79 +49,49 @@ This should bring source and binary files into subdirectories of `/usr/local/go`
 making it possible to access by adding `github.com/tarantool/go-tarantool` in
 the `import {...}` section at the start of any Go program.
 
```

Comment:
This should put the source and binary files in subdirectories of `/usr/local/go`,
so that you can access them by adding `github.com/tarantool/go-tarantool` to the 
`import {...}` section at the start of any Go program.

Reply:
Updated.